### ISSUE TITLE
Moon Orbit Model

### ISSUE THREAD

**Issue by MathieuTuli** [alecjacobson/computer-graphics-shader-pipeline#51]
Is the moon also supposed to be orbiting about itself? It appears as so in the gif.

**Comment by MathieuTuli** [comment]
Further, does it matter whether it orbits CW or CCW around the earth?

**Comment by abhimadan** [comment]
> Is the moon also supposed to be orbiting about itself? 

No. It's confusing in the wireframe gif, look at the planet gif at the bottom - the moon is just revolving around the planet, and isn't rotating about an axis.

> Further, does it matter whether it orbits CW or CCW around the earth?

Either is fine.
